geode-core/build.gradle

```diff
@@ -417,6 +407,28 @@ dependencies {
   }
 }
 
+jmh {
+  duplicateClassesStrategy = 'warn'
+  include = project.hasProperty('jmh.include') ? project.getProperties().get('jmh.include') : '.*'
+  profilers = project.hasProperty('jmh.profilers') ? project.getProperties().get('jmh.profilers').tokenize() : []
+  threads = project.hasProperty('jmh.threads') ? project.getProperties().get('jmh.threads') : null
+}
+
+jmhJar {
+  from {
```

Comment:
Does this change away from using a sub-task for collecting the files make the UP-TO-DATE check work as   expected?

Reply:
Yes, but let me double check it still works.